ehr_analysis2.py

```diff
@@ -2,7 +2,7 @@
 from typing import Union
 
 """Data Parsing"""
-def parse_data(filename):
+def parse_data(filename:str) -> list: 
```

Comment:
List of what? Make type hints as specific as possible.

Reply:
ehr_analysis2.py is just a rename of ehr_analysis.py, when I do rebase, the name changed automatically. This is not a new version.